<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcustomerwindow</class>
 <widget class="QDialog" name="addcustomerwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="streetEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cityEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="statesComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>55</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>AL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AZ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ME</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NJ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ND</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WY</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="zipcodeSpinBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="interestEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Company Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="streetLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Street</string>
   </property>
  </widget>
  <widget class="QLabel" name="cityLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>City</string>
   </property>
  </widget>
  <widget class="QLabel" name="stateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>State</string>
   </property>
  </widget>
  <widget class="QLabel" name="zipcodeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Zipcode</string>
   </property>
  </widget>
  <widget class="QLabel" name="interestLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest</string>
   </property>
  </widget>
  <widget class="QLabel" name="valueLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
